### DISCUSSION TITLE
Plans for Conda Packages?

### DISCUSSION THREAD

**Discussion by alanhdu** [pyg-team/pytorch_geometric#5542] [General]
Right now, the docs say that "# Conda packages are not published for PyTorch 1.12 yet". Is the pyg-team planning to publish those conda packages (and if so, is there a way to chip-in) or is the idea to move away from conda and have everyone use pip?

**Comment by rusty1s** [comment]
Really sorry for the delay in providing conda packages. We had some trouble building `torch-sparse` for PyTorch 1.12 on conda, so this was sadly delayed. Will revisit this ASAP.

**Comment by kmaziarz** [comment]
Is this tied to releasing `pyg` version 2.1.0 through conda? It doesn't seem to be available through the recommended `pyg` channel.

**Reply by rusty1s** [reply]
Yes, it is. Currently, `conda` still only provides `pyg==2.0.*` packages.

**Comment by alanhdu** [comment]
> Really sorry for the delay in providing conda packages. We had some trouble building torch-sparse for PyTorch 1.12 on conda, so this was sadly delayed. Will revisit this ASAP.

Is there anything we can do to help out here? It looks like PyTorch 1.13 is coming out soon-ish, so I'd love to help get the conda packages out!

**Reply by rusty1s** [reply]
Investigating now.

**Comment by rusty1s** [comment]
PyG 2.1 can now (finally) be installed via `conda`, thanks for your patience. Please let me know if it is working.
